pkgs/os-specific/bsd/freebsd/pkgs/bin.nix

```diff
@@ -0,0 +1,92 @@
+{
+  mkDerivation,
+  pkgsBuildBuild,
+  libjail,
+  libmd,
+  libnetbsd,
+  libcapsicum,
+  libcasper,
+  libelf,
+  libxo,
+  libncurses-tinfo,
+  libedit,
+  lib,
+  stdenv,
+  bsdSetupHook,
+  freebsdSetupHook,
+  makeMinimal,
+  install,
+  tsort,
+  lorder,
+  mandoc,
+  groff,
+  byacc,
+  gencat,
+}:
+mkDerivation {
+  pname = "bins";
```

Comment:
Why do we need a single derivation with every binary that FreeBSD has?  It seems weird that we need both sh and csh, for example, when every build environment will already have bash.  Also that groff and mandoc are included, when anywhere else they'd be separately specified inputs.